apptools/persistence/tests/test_two_stage_unpickler.py

```diff
@@ -79,24 +82,19 @@ def test_generic(self):
         s = pickle.dumps(a)
         new_a = pickle.loads(s)
         try:
-            print("\ta.x: %s" % new_a.x)
-            print("\ta.b_ref.y: %s" % new_a.b_ref.y)
-        except Exception as msg:
-            print("\t%s" % "Expected Error".center(75, "*"))
-            print("\t%s" % msg)
-            print("\t%s" % ("*" * 75))
+            new_a.x
+            new_a.b_ref.y
+        except Exception:
+            pass
```

Comment:
Only the second line fails on master:
```
    new_a.b_ref.y
AttributeError: 'B' object has no attribute 'y'
```

While I am not sure whether it was intended that `new_a.x` can be accessed, the test is expecting an exception somewhere from these two lines. In the spirit of keeping the behaviour we know now:
```
# Accessing new_a.x is okay
new_a.x
# Accessing y directly would fail
with self.assertRaisesRegex(
        AttributeError, "'B' object has no attribute 'y'"):
    new_a.b_ref.y
```
(Different styles for checking the exception message exist.)